common/ulsr_hil_client.py

```diff
@@ -309,8 +323,51 @@ def connect_node_to_project(hil_client, node, project):
         hil_client.project.connect(project, node)
         log_info('Node `%s` connected to project `%s`' % (node, project))
     except FailedAPICallException as e:
-        log_error('Unable to connect node `%s` to project `%s`' % (node, project))
+        log_error('Unable to connect node `%s` to project `%s`' %
+                  (node, project))
         raise HILClientFailure(e)
     except ConnectionError as e:
         log_error("connect_node_to_project: Couldn't connect to HIL server.")
         raise HILClientFailure(e)
+
+
+def connect_network(hil_client, node, nic, network, channel):
+    """
+    Connects a single network to specified nic on node.
+
+    Returns a status_id to poll on
+    """
+    try:
+        response = hil_client.node.connect_network(
+            node, nic, network, 'vlan/native')
+        status_id = response['status_id']
+        log_info('Node `%s` succesfully connected to network `%s`' %
+                 (node, network))
+        return status_id
+    except FailedAPICallException as e:
```

Comment:
For these functions that have the two `except`s at the end, was it your plan for unexpected errors to just show their output?

Reply:
Yes, I am not catching unexpected errors because we don't know how to handle the unknown. If/when an unexpected error is raised, that will just stop the code and the message will be loud and clear (will be useful for debugging).

Reply:
Sounds good